paper/paper.md

```diff
@@ -0,0 +1,74 @@
+---
+title: 'scikit-COSMO: A Toolbox of Machine Learning Methods for Materials Science'
+tags:
+    - Python
+authors:
+    - name: Rose K. Cersonsky^[\url{mailto:rose.cersonsky@epfl.ch}]
+      orcid: 0000-0003-4515-3441
+      affiliation: 1
+    - name: Guillaume Fraux
+      orcid: 0000-0003-4824-6512
+      affiliation: 1
+    - name: Sergei Kliavinek
+      orcid: 0000-0001-8326-325X
+      affiliation: 1
+    - name: Alexander Goscinski
+      orcid: 0000-0001-8076-215X
+      affiliation: 1
+    - name: Benjamin A. Helfrecht
+      orcid: 0000-0002-2260-7183
+      affiliation: 1
+    - name: Michele Ceriotti ^[\url{mailto:michele.ceriotti@epfl.ch}]
+      orcid: 0000-0003-2571-2832
+      affiliation: 1
+affiliations:
+    - name: Laboratory of Computational Science and Modeling (COSMO), École Polytechnique Fédérale de Lausanne(EPFL), Lausanne, Switzerland
+      index: 1
+date: 21 Jun 2021
+bibliography: paper.bib
+---
+
+# Summary
+
+`scikit-COSMO` (`skcosmo`) focuses on machine learning methods that are actively used, for example, in the field of computational materials science and chemistry. Currently, many
```

Comment:
This sentence feels strange to me. It seems like there should be some kind of "by opposition to" or some other clarification. I understand this tries to say that skcosmo is not only for materials science, right?

How about something like this?
```suggestion
`scikit-COSMO` (`skcosmo`) contains a collection of machine learning methods initially developed for computational materials science and chemistry applications, but useful in a wide range of fields. Currently, many
```

Reply:
do we need a summary? This seems to be well explained in the next section